content/src/logic/database-queries/snapshots-queries.ts

```diff
@@ -33,3 +35,182 @@ export async function* streamActiveDeployments(
     }
   }
 }
+
+export async function* streamActiveDeploymentsInTimeRange(
+  components: Pick<AppComponents, 'database'>,
+  timeRange: TimeRange
+): AsyncIterable<DeploymentWithAuthChain> {
+  // IT IS IMPORTANT THAT THIS QUERY NEVER CHANGES. ORDER IS NOT GUARANTEED  `
```

Comment:
why can't we guarantee the order?

Reply:
we can but the query could take more time